NEON_Products.csv

```diff
@@ -0,0 +1,25 @@
+NEON Product,NEON Product ID,,Sampling level

```

Comment:
Is this the 'master' list, or is this an csv export of a list that actually lives elsewhere (e.g. Google Sheet, Dropbox)?  I ask because it's always better to have a single master list than multiple copies that have to be kept in sync.